AspNetCore.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26124.0
+# Visual Studio Version 16
```

Comment:
@Pilchie @MackinnonBuck tried to add a project using VS and it changed quite a few GUIDs and version numbers. I think he had to use VS (vs dotnet sln add) to create solution folders. Any idea why VS is producing so much diff?

Reply:
Ah - one of those guids is "original C# project guid, so please autodetect whether to use the old project system, or the new SDK project system", and the other is "force use of the new project system".

I forget which is which, but I *think* the `9A19103F` is the original guid, which VS seems to prefer.

It seems that `dotnet sln add` uses the `FAE04EC0` one to force to the new project system though.

@davkean - does that match your understanding?

If so, we should probably just update them all, and file an issue on `dotnet/sdk` to change `dotnet sln add` to use the `9A19103F`